target/linux/layerscape/armv8_32b/target.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2016 Jiang Yutang <jiangyutang1978@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+ARCH:=arm
+BOARDNAME:=ARMv8 32-bit based boards
+CPU_TYPE:=cortex-a9
```

Comment:
As the layerscape 32 biot target is not so popular I would suggest choosing something which is already used by some other target, see here: https://downloads.lede-project.org/snapshots/packages/

I would suggest this:
CPU_TYPE:=cortex-a15
CPU_SUBTYPE:=neon-vfpv4 
This is probably a better fit than the A9